templates/ruby/pkgs/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, bundlerEnv, ruby }:
+let
+  # the magic which will include gemset.nix
+  gems = bundlerEnv {
+    name = "my-package-env";
+    inherit ruby;
+    gemfile = ../Gemfile;
+    lockfile = ../Gemfile.lock;
+    gemset = ../gemset.nix;
+  };
+in stdenv.mkDerivation rec {
+  name = "my-package";
+  src = ../.;
+  buildInputs = [gems ruby];
+  installPhase = ''
+    mkdir -p $out/bin  $out/share/${name}
+    cp -r * $out/share/${name}
+    bin=$out/bin/${name}
+# we are using bundle exec to start in the bundled environment
```

Comment:
Not sure if this indentation is intentional?

Reply:
@mkenigs thanks I will clean this up as soon as I am in front of dev computer again